wondrous-app/components/Common/SnackbarAlert/index.tsx

```diff
@@ -1,9 +1,11 @@
-import { createContext, useEffect, useState } from 'react';
+import { createContext, useContext, useEffect, useState } from 'react';
 import { useIsMobile } from 'utils/hooks';
 import { StyledAlert, StyledSnackbar } from './styles';
 
 export const SnackbarAlertContext = createContext(null);
 
+export const useSnackbarAlert = () => useContext(SnackbarAlertContext);
```

Comment:
no need for this imo
  const { setSnackbarAlertOpen, setSnackbarAlertMessage, setSnackbarAlertSeverity } = useContext(SnackbarAlertContext); 
  directly